### PR TITLE
feat: add Intercom chat service

### PR DESCRIPTION
**Pull request by JulioJu**
Following https://github.com/AmauriC/tarteaucitron.js/pull/682

### CODE REVIEW

File: tarteaucitron.services.js [hunk 1]
```diff
@@ -2397,6 +2397,52 @@ tarteaucitron.services.purechat = {
...
+            };
+            i.q = [];
+            i.c = function (args) {
+                i.q.push(args);
+            };
+            w.Intercom = i;
+            tarteaucitron.addScript(
+                "https://widget.intercom.io/widget/" + tarteaucitron.user.intercomKey,
+                "",
+                intercomChatEnable(),
```
Comment (JulioJu) [inline tarteaucitron.services.js:2434]: Fix comment from @AmauriC at https://github.com/AmauriC/tarteaucitron.js/pull/682#discussion_r622871085

File: tarteaucitron.services.js [hunk 2]
```diff
@@ -2397,6 +2397,52 @@ tarteaucitron.services.purechat = {
...
+            );
+        }
+    },
+    "fallback": function () {
+        "use strict";
+        var id = "intercomChat";
+        tarteaucitron.fallback(["intercom-chat"], tarteaucitron.engage(id));
+        intercomChatDisable()
```
Comment (AmauriC) [inline tarteaucitron.services.js:2442]: This function will thrown an error because the intercom script will not be loaded.

Comment (JulioJu) [reply tarteaucitron.services.js:2538]: @AmauriC sorry for the dalay. 

I've corrected this. Could you check again please ?

Comment (AmauriC) [reply tarteaucitron.services.js:2538]: Hey,
I still see the `intercomChatDisable()` function 🧐

**Comment by JulioJu** [comment]
@AmauriC sorry I have not seen your answer. Could you check if it's correct now ?

File: tarteaucitron.services.js [hunk 3]
```diff
@@ -2493,6 +2493,67 @@ tarteaucitron.services.purechat = {
...
+    "uri": "https://www.intercom.com/",
+    "needConsent": true,
+    "cookies": [
+        "intercom-id-" + tarteaucitron.user.ingercomKey,
+        "intercom-session-" + tarteaucitron.user.ingercomKey,
+    ],
+    "readmoreLink": "https://www.intercom.com/legal/privacy",
+    "js": function () {
+        window.intercomSettings = {
+            app_id: tarteaucitron.user.ingercomKey,
```
Comment (AmauriC) [inline tarteaucitron.services.js:2510]: `ingercomKey` à la place de `intercomKey`

**Review by AmauriC — CHANGES_REQUESTED** [review]
Dès que c'est modifié je valide : )

File: tarteaucitron.services.js (hunk 3 above)
Comment (JulioJu) [reply tarteaucitron.services.js]: Oupse, corrigé !

**Pull request merged**